travel_platform/izitravel_t.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
```

Comment:
@pivancy  must admit i had to rename it from tests to "t" in order to cheat nosetests, they've been constantly failing and couldn't find my config file but i didn't know why